content/post/mnist-example.md

```diff
@@ -0,0 +1,148 @@
++++
+title = "A Scikit-learn pipeline in Wallaroo"
+draft = false
+date = 2017-11-16T00:00:00Z
+tags = [
+    "wallaroo",
+    "machine learning",
+    "python",
+    "example",
+    "tutorial"
+]
+categories = [
+    "Machine Learning"
+]
+description = "Creating an inference pipeline with MNIST"
+author = "amosca"
++++
+
+Nowadays, many applications with streaming data are either applying machine learning or have a very good use case for it. In this example, we will explore how we can build a sample pipeline to classify images from the [MNIST dataset](ihttp://yann.lecun.com/exdb/mnist/), using PCA and a regression tree in scikit-learn. As a prerequisite, you should be familiar with some notions of machine learning, and some notions about Wallaroo pipelines.
```

Comment:
I think you need to explain what the MNIST dataset is.

I think you should explain the basics of some of these notions.